cheatsheets/DotNet_Security_Cheat_Sheet.md

```diff
@@ -219,11 +219,13 @@ HttpContext.Current.Response.Headers.Remove("Server");
 
 ASP.NET MVC (Model–View–Controller) is a contemporary web application framework that uses more standardized HTTP communication than the Web Forms postback model. 
 
-The OWASP Top 10 lists the most prevalent and dangerous threats to web security in the world today and is reviewed every 3 years. 
+The OWASP Top 10 2017 lists the most prevalent and dangerous threats to web security in the world today and is reviewed every 3 years. 
 
 This section is based on this. Your approach to securing your web application should be to start at the top threat A1 below and work down, this will ensure that any time spent on security will be spent most effectively spent and cover the top threats first and lesser threats afterwards. After covering the top 10 it is generally advisable to assess for other threats or get a professionally completed Penetration Test.
 
-## A1 SQL Injection
+## A1 Injection
+
+### SQL Injection
```

Comment:
It miss the info link to the [Query Parameterization Cheat Sheet](https://github.com/OWASP/CheatSheetSeries/blob/master/cheatsheets/Query_Parameterization_Cheat_Sheet.md) like it is done in the LDAP injection.

Reply:
Still missing when I take the content of your [fork](https://github.com/knadt/CheatSheetSeries/blob/master/cheatsheets/DotNet_Security_Cheat_Sheet.md)

Reply:
Updated Line 232

Reply:
<img width="867" alt="Screenshot 2019-08-16 at 07 53 57" src="https://user-images.githubusercontent.com/16526578/63146392-174b0600-bffb-11e9-93a8-b7eb17434b2a.png">